.rubocop.yml

```diff
@@ -217,6 +217,7 @@ Layout/SpaceAroundOperators:
     - '**/spec/actions_specs/xcodebuild_spec.rb'
 
 AllCops:
+  TargetRubyVersion: 2.4
```

Comment:
According to the official document, it should be set minimum support version.

https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version


```
If .ruby-version exists in the directory RuboCop is invoked in, RuboCop will use the version specified by it. Otherwise, users may let RuboCop know the oldest version of Ruby which your project supports with:
```

So we should set this to `2.0`. 👍